packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -418,7 +418,7 @@ class IrisGridUtils {
   }
 
   static getInputFiltersForColumns(
-    columns: readonly DhType.Column[],
+    columns: readonly { name: string; type: string }[],
```

Comment:
Hrm. Another one of these types, though we don't want `iris-grid` depending on `dashboard-core-plugins`.
We've got out `@deephaven/filters` package - maybe makes sense to pull the input filter type definitions down and use them in both `iris-grid`/`dashboard-core-plugins`?

Reply:
This util doesn't even need to be in `IrisGridUtils`. It's only used by the panel and the new hook which are both in `dashboard-core-plugins`. I don't see it used anywhere in enterprise

For relatively simple types like this, I'm not opposed to just having it as an inline literal. The util doesn't actually care that it's an input filter column or a table column or a dropdown filter column. It cares that it has those 2 props and all those other types satisfy that constraint

Originally I had all of these as just `dh.Column[]`, but that is not the case when using it within charts. Chart has a `FilterColumnMap` type that has values `{ name: string; type: string }`

Reply:
Yea it probably doesn't really make sense in IrisGridUtils. But no need to make that a breaking change here.